File: sparkGmall/sparkmall-common/src/main/scala/com/shangbaishuyao/utils/PropertiesUtil.scala
package com.shangbaishuyao.utils

import java.io.InputStreamReader
import java.util.Properties

/**
 * Desc: 配置信息工具类 <br/>
 * create by shangbaishuyao on 2021/3/9
 * @Author: 上白书妖
 * @Date: 14:35 2021/3/9
 */
object PropertiesUtil {

  def load(propertiesName:String): Properties ={

    val prop=new Properties()
    prop.load(new InputStreamReader(Thread.currentThread().getContextClassLoader.getResourceAsStream(propertiesName) , "UTF-8"))
    prop
  }

}

File: Gmall/Gmall-common/src/main/java/com/shangbaishuyao/utils/PropertiesUtil.java
package com.shangbaishuyao.utils;

import java.io.IOException;
import java.util.Properties;
/**
 * Desc: 通过路径加载配置文件 <br/>
 * create by shangbaishuyao on 2021/3/14
 * @Author: 上白书妖
 * @Date: 18:08 2021/3/14
 */
public class PropertiesUtil {
    public static Properties load(String path) {
        Properties properties = new Properties();
        try {
            properties.load(ClassLoader.getSystemResourceAsStream(path));
        } catch (IOException e) {
            e.printStackTrace();
        }
        return properties;
    }
}
